src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,69 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
```

Comment:
Is there still a point in having that `AnimationClass` defined for each type? It looks like this code can be unified and just be moved here instead of having it duplicated for each animation type

Reply:
Each animation has different state